src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -233,7 +233,10 @@ class ExposureSubmissionCoordinator: NSObject, ExposureSubmissionCoordinating, R
 	}
 	
 	private func createTestResultViewController(with testResult: TestResult) -> ExposureSubmissionTestResultViewController {
-		updateStoreWithKeySubmissionMetadataDefaultValues()
+		// store is only initialized when a positive test result is received
+		if testResult == .positive {
+			updateStoreWithKeySubmissionMetadataDefaultValues()
+		}
```

Comment:
when the store is only initialized when a positive test result is received, what happens if there is a negative result? Do we track then nothing?

Reply:
Yes. Nothing. 

Tech Specs:
A data structure to represent the values for PPAKeySubmissionMetadata is created when a positive test result is received.